core/modules/widgets/image.js

```diff
@@ -149,13 +138,27 @@ ImageWidget.prototype.execute = function() {
 Selectively refreshes the widget if needed. Returns true if the widget or any of its children needed re-rendering
 */
 ImageWidget.prototype.refresh = function(changedTiddlers) {
-	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.source || changedAttributes.width || changedAttributes.height || changedAttributes["class"] || changedAttributes.usemap || changedAttributes.tooltip || changedTiddlers[this.imageSource]) {
+	var changedAttributes = this.computeAttributes(),
+		hasChangedAttributes = $tw.utils.count(changedAttributes) > 0,
+		alwaysRerender = [
+			"source",
+			"loading",
+			"class"
+		],
+		attributesNeedRerender = alwaysRerender.some(key => changedAttributes[key]);
```

Comment:
At the moment, we have no decision if ES6 arrow functions are allowed. @Jermolene ??

Reply:
I'm in the process of backporting this from a branch intended for v5.4.0,
which largely involves reverting to older JavaScript syntax, which is why
this is still marked as a draft.